frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -166,6 +168,10 @@ export default {
         const payload = { model: this.model, id: this.metaboliteId };
         await this.$store.dispatch('metabolites/getMetaboliteData', payload);
         this.componentNotFound = false;
+        if (this.metabolite.externalDbs.ChEBI) {
+          const { data } = await axios.get(`https://www.ebi.ac.uk/chebi/displayImage.do?defaultImage=true&imageIndex=0&chebiId=${this.metabolite.externalDbs.ChEBI[0].id.slice(6)}`);
```

Comment:
I think it will be better to define the url for ChEBI as a variable so that we do not need to change it at multiple places when this url is changed.

Reply:
Ah ofc, great feedback!